test/node/double.test.ts

```diff
@@ -40,159 +47,174 @@ describe('BSON Double Precision', function () {
         },
         {
           // Reference: https://docs.oracle.com/cd/E19957-01/806-3568/ncg_math.html
           // min positive normal number
-          title:
-            'returns "2.2250738585072014e-308" when input is a number the minimum positive normal value',
+          title: 'preserves the minimum possible stringified normal value',
           input: '2.2250738585072014e-308',
           output: { $numberDouble: '2.2250738585072014e-308' }
         },
         {
           // max subnormal number
-          title:
-            'returns "2.225073858507201e-308" when input is a number the maximum positive subnormal value',
+          title: 'preserves the max possible stringified subnormal value',
           input: '2.225073858507201e-308',
           output: { $numberDouble: '2.225073858507201e-308' }
         },
         {
           // min positive subnormal number (NOTE: JS does not output same input string, but numeric values are equal)
-          title: 'returns "5e-324" when input is a number the minimum positive subnormal value',
+          title: 'simplifies but preserves the minimum possible subnormal value',
           input: '4.9406564584124654e-324',
           output: { $numberDouble: '5e-324' }
-        },
-        {
-          // https://262.ecma-international.org/13.0/#sec-number.prototype.tofixed
-          // Note: calling toString on this integer returns 1000000000000000100, so toFixed is more precise
-          // This test asserts we do not change _current_ behavior, however preserving this value is not
-          // something that is possible in BSON, if a future version of this library were to emit
-          // "1000000000000000100.0" instead, it would not be incorrect from a BSON/MongoDB/Double precision perspective,
-          //  it would just constrain the string output to what is possible with 8 bytes of floating point precision
-          title:
-            'returns "1000000000000000128.0" when input is an int-like number beyond 8-byte double precision',
-          input: '1000000000000000128',
-          output: { $numberDouble: '1000000000000000128.0' }
         }
       ];
 
-      for (const test of tests) {
-        const input = test.input;
-        const output = test.output;
-        const title = test.title;
-        it(title, () => {
-          const inputAsDouble = new Double(input);
-          expect(inputAsDouble.toExtendedJSON({ relaxed: false })).to.deep.equal(output);
-          if (!Number.isNaN(inputAsDouble.value)) {
-            expect(Number(inputAsDouble.toExtendedJSON({ relaxed: false }).$numberDouble)).to.equal(
-              inputAsDouble.value
-            );
-          }
-        });
+      context('returns a stringified value that', () => {
+        for (const test of tests) {
+          const input = test.input;
+          const output = test.output;
+          const title = test.title;
+          it(title, () => {
+            const inputAsDouble = new Double(input);
```

Comment:
This isn't exactly what I was looking for, I don't think I read the tests thoroughly enough on my first pass.  

The context should break apart the test classes, not group all test classes.  
The tests should not contain a "title" property, instead they should describe the class of test cases they're covering (i.e., a context) and that should be listed in the context blocks.  

Here's a before (with the current test structure) and after (for the first four test cases).  I think the latter is much easier to understand

<img width="794" alt="Screen Shot 2023-01-13 at 11 31 43 AM" src="https://user-images.githubusercontent.com/23407842/212370855-701dcf62-5ac1-45d5-adfa-54cdd2d0e8b9.png">
<img width="586" alt="Screen Shot 2023-01-13 at 11 35 50 AM" src="https://user-images.githubusercontent.com/23407842/212371714-c2a0386c-8680-488f-b06e-b6958798c356.png">

Reply:
<img width="633" alt="Screen Shot 2023-01-13 at 11 38 46 AM" src="https://user-images.githubusercontent.com/23407842/212372288-4f9246e7-23b7-407b-9fd5-17ce92ef6833.png">
here's my quick code changes for reference

Reply:
"Stringifies to EJSON correctly" what's correctly mean in that context?

Reply:
that was an example

if the title there is important, that could also be included in the tests (not my preference, why would we include one title but not the other) but another title could be generated

`stringifies 0 to { $numberDouble: 0.0 }` for example

Reply:
Sure, updated